internal/upstreamldap/upstreamldap_test.go

```diff
@@ -638,8 +638,8 @@ func TestEndUserAuthentication(t *testing.T) {
 			username: testUpstreamUsername,
 			password: testUpstreamPassword,
 			providerConfig: providerConfig(func(p *ProviderConfig) {
```

Comment:
I do not see a unit test for the `internal/upstreamldap/upstreamldap.go` change?